### PR TITLE
chore(macros/JsSidebar): update zh-CN translation

### PR DESCRIPTION
**Pull request by jasonren0403**
## Summary

Update strings in JsSidebar to improve translation

part of https://github.com/mdn/translated-content/issues/13881

cc: @yin1999 @awxiaoxian2020

### CODE REVIEW

File: kumascript/macros/JsSidebar.ejs [hunk 1]
```diff
@@ -209,7 +209,7 @@ var text = mdn.localStringMap({
     'Data_structures': 'JavaScript 数据结构',
     'Equality': '如何正确判断相等性',
     'Closures': '闭包',
-    'Advanced': '高级',
+    'Advanced': '进阶教程',
```
Comment (yin1999) [inline kumascript/macros/JsSidebar.ejs:212]: How about "进阶", as this is not a part of learning module.

Comment (awxiaoxian2020) [reply kumascript/macros/JsSidebar.ejs:212]: @jasonren0403 I agree with @yin1999. If you think it's strange that we use `高级` in sidebar or title in Chinese situations, I think it's not about the translations, it's about organizations that we can't change it in short time or we shouldn't change it. So I think we can use the previous translation.

Comment (jasonren0403) [reply kumascript/macros/JsSidebar.ejs:212]:
```suggestion
    'Advanced': '进阶',
```

File: kumascript/macros/JsSidebar.ejs [hunk 2]
```diff
@@ -200,7 +200,7 @@ var text = mdn.localStringMap({
     'Complete_beginners': '快速入门',
     'Basics': 'JavaScript 基础知识',
     'First_steps': 'JavaScript 第一步',
-    'Building_blocks': '创建 JavaScript 代码块',
+    'Building_blocks': '构建 JavaScript 代码块',
     'Intermediate': '中级教程',
```
Comment (jasonren0403) [inline kumascript/macros/JsSidebar.ejs:204]:
```suggestion
    'Intermediate': '中级',
```

File: kumascript/macros/JsSidebar.ejs (hunk 1 above)
Comment (jasonren0403) [reply kumascript/macros/JsSidebar.ejs:212]: I choose to remove '教程' because they are already under 'tutorials' 🤔

Comment (yin1999) [reply kumascript/macros/JsSidebar.ejs:212]: +1

**Comment by awxiaoxian2020** [comment]
No objection.

**Review by yin1999 — APPROVED** [review]
LGTM :+1:

**Review by caugner — APPROVED** [review]

**Pull request merged**